webapp/packages/plugin-navigation-tree/src/index.ts

```diff
@@ -57,6 +57,13 @@ export * from './TreeNew/useTreeFilter.js';
 export * from './TreeNew/useTreeSelection.js';
 export * from './TreeNew/INode.js';
 export * from './TreeNew/NodeEmptyPlaceholderComponent.js';
+export * from './TreeNew/TreeToolbar.js';
```

Comment:
We should wrap it in lazy component as we do for NodeControlLazy NodeLazy and TreeLazy